lib/libc/arcmwdt/arcmwdt-string.c

```diff
@@ -0,0 +1,30 @@
+/* string.c - common string routines */
+
+/*
+ * Copyright (c) 2014 Wind River Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <string.h>
+#include <stdint.h>
+#include <sys/types.h>
+
+/**
+ *
+ * @brief Get fixed-size string length
+ *
+ * @return number of bytes in fixed-size string <s>
+ */
+
+size_t strnlen(const char *s, size_t maxlen)
```

Comment:
No strnlen in MW libc? maybe file a enhancement request for that...
I do see strnlen_s is part of MW, use that instead?

Reply:
Fixed. Now we simply call strnlen_s in strnlen.